docs/GroupDefs.hpp

```diff
@@ -1229,3 +1229,8 @@ then typically look as follows:
  * \defgroup UtilitiesGroup Utilities
  * \brief A collection of useful classes, functions and metafunctions.
  */
+
+/*!
+ * \defgroup VariableFixingGroup Variable Fixing
+ * \brief A collection of different variable fixers ranging in sophistication.
```

Comment:
It would be good to have the long description explain what variable fixing is and why.